### PR TITLE
GEODE-6850: change int stats to long

### PR DESCRIPTION
**Pull request by dschneider-pivotal**
The "int" methods on the statistics interfaces have been deprecated.
The implementation of these deprecated "int" methods now use the "long" methods.
The only place this could be visible to a user is if they call Statistics.get(String) or Statistics.get(StatisticDescriptor). Both of these methods return a "Number" instance.
These methods continue to do this and existing code will work fine if it does not try to downcast the Number to a subclass. But for "int" stats, these methods used to return an "Integer" instance and now return a "Long" instance.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/StatisticsTypeFactory.java [hunk 1]
```diff
@@ -156,6 +161,9 @@ StatisticDescriptor createDoubleCounter(String name, String description, String
   /**
    * Creates and returns an int gauge {@link StatisticDescriptor} with the given <code>name</code>,
    * <code>description</code>, <code>largerBetter</code>, and <code>units</code>.
+   *
+   * @deprecated as of Geode 1.10, use {@link #createLongGauge(String, String, String, boolean)}
+   *             instead
    */
   StatisticDescriptor createIntGauge(String name, String description, String units,
```
Comment (pivotal-jbarrett) [inline geode-core/src/main/java/org/apache/geode/StatisticsTypeFactory.java]: Can you use the method annotation `@Deprecated` too.

Comment (dschneider-pivotal) [reply geode-core/src/main/java/org/apache/geode/StatisticsTypeFactory.java]: done

**Comment by dschneider-pivotal** [comment]
I confirmed that the StressNewTestOpenJDK11 failures were caused by updating the stat code in two cq query dunit tests. Those tests run very slowly so when the Stressor runs them a bunch of times it takes longer than 2.5 hours at which point StressNewTestOpenJDK11 times out.

**Comment by pivotal-jbarrett** [comment]
@dschneider-pivotal then I think you can safely ignore it.

**Pull request merged**